### PR TITLE
✨ Implementation of JSON Server

### PR DESCRIPTION
**Pull request by Sina-Sche**
- [x] Install JSON Server

- [x] write first API Request  (getProducts())

- [x] implement API Request to display Products on HomeScreen

### CODE REVIEW

**Review by BelaM80 — COMMENTED** [review]
LGTM! 🎉

File: client/package.json [hunk 1]
```diff
@@ -27,7 +28,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "storybook": "start-storybook -p 3600 -s public",
-    "build-storybook": "build-storybook -s public"
+    "build-storybook": "build-storybook -s public",
+    "api": "json-server --watch db.json --port 4001"
```
Comment (phlgr) [inline client/package.json:32]: You can remove this script as you don't need it anymore

File: client/src/components/ProductList.js [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
 import ProductPreview from "./ProductPreview";
-import Hairbrush from "../assets/images/hairbrush.jpg";
+import { useState, useEffect } from "react";
+import { getProducts } from "../utils/api";
 
 const ListContainer = styled.ul`
   display: grid;
   grid-template-columns: 1fr 1fr;
   list-style: none;
+  gap: 10px;
```
Comment (phlgr) [inline client/src/components/ProductList.js:10]: Didn't know that property. 🎉

File: client/src/components/ProductPreview.js [hunk 3]
```diff
@@ -9,34 +9,38 @@ const Container = styled.div`
...
   return (
     <Container primary={primary}>
       {primary ? <h2>Our Pick of the Day</h2> : <></>}
       <Image src={src} alt={alt} size={size} />
-      <TextContainer title={title} primary={true} />
+      {size === "small" ? (
+        <></>
+      ) : (
+        <TextContainer title={title} primary={true} />
+      )}
```
Comment (phlgr) [inline client/src/components/ProductPreview.js:36]:
```suggestion
      {!size === "small" ? 
        <TextContainer title={title} primary={true} />
      }
```

File: client/src/db.json [hunk 4]
```diff
@@ -0,0 +1,279 @@
+{
```
Comment (phlgr) [inline client/src/db.json:1]: db.json should be located in the root folder. This is not needed

File: package.json [hunk 5]
```diff
@@ -13,7 +13,8 @@
     "server": "nodemon server.js",
     "client": "cd client && npm start",
     "build": "cd client && npm run build && npm run build-storybook",
-    "storybook": "cd client && npm run storybook"
+    "storybook": "cd client && npm run storybook",
+    "api": "json-server --watch db.json --port 7000"
```
Comment (phlgr) [inline package.json:17]: This script is no longer needed

**Review by phlgr — CHANGES_REQUESTED** [review]

File: client/package.json (hunk 1 above)
Comment (Sina-Sche) [reply client/package.json:32]: Ah, that makes sense and probably helps solve my deployment error with Heroku :) Thanks!

File: client/src/db.json (hunk 4 above)
Comment (Sina-Sche) [reply client/src/db.json:1]: Oh, that was still a "left-over" from when I had it installed in the client 🙈

File: client/src/components/ProductPreview.js (hunk 3 above)
Comment (Sina-Sche) [reply client/src/components/ProductPreview.js]: It did not work without the `:` 
I kept getting an error: `Unexpected token expected : : `

**Pull request merged**